### PR TITLE
add e2esku tests

### PR DESCRIPTION
**Pull request by markwolff**
Adds some e2e tests, misc bug fixes

### CODE REVIEW

File: AISKU/Initialization.ts [hunk 1]
```diff
@@ -114,10 +130,16 @@ export class Initialization {
...
-                appInsightsInstance.core.getTransmissionControl().flush(true);
+                appInsightsInstance.core.getTransmissionControls().forEach(queues => {
+                    queues.forEach(channel => channel.flush(true));
+                });
+                
                 // Back up the current session to local storage
                 // This lets us close expired sessions after the cookies themselves expire
-                this.properties._sessionManager.backup();
+                // Todo: move this against interface behavior
+                if (this.core.extensions["AppInsightsPropertiesPlugin"] && this.core.extensions["AppInsightsPropertiesPlugin"]._sessionManager) {
```
Comment (jpiyali) [inline AISKU/Initialization.ts]: format

File: ApplicationInsights/JavaScriptSDK/Telemetry/PageViewManager.ts [hunk 2]
```diff
@@ -158,6 +165,6 @@ export class PageViewManager {
                     "trackPageView failed on page load calculation: " + Util.getExceptionName(e),
                     { exception: Util.dump(e) });
             }
-        }, 100);
+        }).bind(this), 100);
```
Comment (jpiyali) [inline ApplicationInsights/JavaScriptSDK/Telemetry/PageViewManager.ts]: take out

File: AISKU/Init.ts [hunk 3]
```diff
@@ -1,92 +1,45 @@
...
     // E2E sku on load initializes core and pipeline using snippet as input for configuration
-    
+    var aiName;
     if (typeof window !== "undefined" && typeof JSON !== "undefined") {
         // get snippet or initialize to an empty object
 
-        // appinsightsvnext should not conflict if page uses existing sdk for a layer of instrumentation
-        var aiName = "appInsightsvNext";
+        // get sdk instance name should not conflict if page uses existing sdk for a layer of instrumentation
+        aiName = window["appInsightsSDK"];
```
Comment (markwolff) [inline AISKU/Init.ts]: do we want to take this as an arg?

File: AISKU/package.json [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+        "webpack-command": "^0.4.1",
+        "source-map-loader": "^0.2.3",
+        "ts-loader": "^4.4.2"
+    },
+    "publishConfig": {
+        "registry": "https://mseng.pkgs.visualstudio.com/_packaging/ApplicationInsights-Team/npm/registry"
+    },
+    "dependencies": {
+        "applicationinsights-analytics-js": "file:../ApplicationInsights/amd",
+        "applicationinsights-channel-js": "file:../../applicationinsights-channel-js/amd/",
```
Comment (markwolff) [inline AISKU/package.json]: delete this file in PR?

Comment (jpiyali) [reply AISKU/package.json]: yes

File: AISKU/Init.ts (hunk 3 above)
Comment (jpiyali) [reply AISKU/Init.ts]: yes

Comment (jpiyali) [reply AISKU/Init.ts]: ignore my comment, this is fine.

**Pull request merged**